css/style.css

```diff
@@ -16,4 +27,441 @@ body {
+    margin: 0;
+}
+
+.hero h2 {
+    font-weight: 200;
+    font-size: 35px;
+    text-align: center;
+}
+
+.hero h5 {
+    margin-top: -1rem;
+    margin-bottom: 2rem;
+    font-weight: 50;
+    font-size: 20px;
+}
+
+.hero button {
+    background-color: var(--orange);
+    border: none;
+    padding: .8rem 1rem;
+    border-radius: 5px;
+}
+.hero button:hover {
+    background-color: var(--grey-dark);
+}
+
+.hero button a {
+    text-decoration: none;
+    color: var(--white);
+}
+
+/* Second section */
+section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 4rem;
+    margin-bottom: 3rem;
+}
+
+section h2 {
+    font-weight: 200;
+    font-size: 35px;
+}
+
+article {
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    
+    
+}
+
+article h5 {
+    font-weight: 200;
+    font-size: 20px;
+    
+}
+
+article img {
+    width: 120px;
+    height: 120px;
+}
+
+.context {
+    margin: 3rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1.5rem;
+    
+}
+
+/* footer */
+footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    border-top: 1px solid var(--grey);
+    margin: 0 9rem;
+    padding: 2rem 0;
+    gap: 1rem;
+}
+
+footer ul {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+}
+footer ul li a {
+    display: block;
+    border: 1px solid var(--grey);
+    padding: 1rem;
+    border-radius: 50%;
+    margin: 10px;
+}
+
+footer ul li a img{
+    max-width: 1rem;
+    max-height: 1rem;
+} 
+
+footer h2 {
+    font-weight: 400;
+}
+
+footer h5 {
+    font-weight: 200;
+    color: var(--grey);
+}
+/* level 2 */
+/* container */
+.continer_ads{
+    display: grid;
+    grid-template-columns:1.5fr 2fr ;
+    grid-template-rows:auto;
+}
+.container_photo{
+    background-image: url(/level-2/homepage-feature.png);
+    background-size: cover;
+    height: 100%;
+    background-repeat: none;
+}
+.container_context{
+    padding: 8rem;
+}
+
+.store-main {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+/* section one */
+.form-section {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    padding: 2rem;
+    gap: 30px;
+}
+/* form */
+form {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+
+/* form title */
+.form-section h1 {
+    color: var(--orange);
+}
+
+/* form classes */
+label {
+    font-size: 12px;
+    font-weight: 400;
+    color: var(--grey-dark);
+}
+
+.fieldset {
+    display: flex;
+    flex-direction: row;
+    align-items: stretch;
+    flex-wrap: wrap;
+    border: none;
+    gap: 30px;
+}
+
+.flex {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    gap: 5px;
+}
+
+/* input style */
+input[type="text"] {
+    height: 30px;
+    padding: 2px;
+}
+
+input:focus {
+    outline-color: var(--orange) !important;
```

Comment:
Good to see that you've put the work in to make sure that your submission is as accessible as possible!